Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Debug/RayTracingDebugFeatureProcessorInterface.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Atom/Feature/Debug/RayTracingDebugSettingsInterface.h>
+#include <Atom/RPI.Public/FeatureProcessor.h>
+
+namespace AZ::Render
+{
+    class RayTracingDebugFeatureProcessorInterface : public RPI::FeatureProcessor
```

Comment:
Recommend adding some comments explaining the purpose of this interface.

Reply:
Added comments for class and functions